src/components/panel/panel.spec.js

```diff
@@ -63,69 +64,121 @@ describe('$mdPanel', function() {
   });
 
   it('should create and open a basic panel', function() {
-    var template = '<div id="panel">Hello World!</div>';
+    var template = '<div>Hello World!</div>';
     var config = { template: template };
 
     panelRef = $mdPanel.open(config);
     $rootScope.$apply();
 
-    expect('#panel').toExist();
+    expect(PANEL_EL).toExist();
     expect(panelRef.isOpen).toEqual(true);
 
     panelRef.close();
     $rootScope.$apply();
 
-    expect('#panel').not.toExist();
+    expect(PANEL_EL).not.toExist();
     expect(panelRef.isOpen).toEqual(false);
   });
 
   it('should add and remove a panel from the DOM', function() {
-    var template = '<div id="panel">Hello World!</div>';
+    var template = '<div>Hello World!</div>';
     var config = { template: template };
 
     panelRef = $mdPanel.create(config);
 
-    expect('#panel').not.toExist();
+    expect(PANEL_EL).not.toExist();
 
     panelRef.open();
     $rootScope.$apply();
 
-    expect('#panel').toExist();
+    expect(PANEL_EL).toExist();
 
     panelRef.close();
     $rootScope.$apply();
 
-    expect('#panel').not.toExist();
+    expect(PANEL_EL).not.toExist();
   });
 
-  it('should attach panel to a specific element', function() {
-    var parentEl = document.createElement('div');
-    parentEl.id = 'parent';
-    attachToBody(parentEl);
+  describe('config options:', function() {
+    it('should attach panel to a specific element', function() {
+      var parentEl = document.createElement('div');
+      parentEl.id = 'parent';
+      attachToBody(parentEl);
 
-    var template = '<div id="panel">Hello World!</div>';
-    var config = {
-      attachTo: angular.element(parentEl),
-      template: template
-    };
+      var template = '<div>Hello World!</div>';
+      var config = {
+        attachTo: angular.element(parentEl),
+        template: template
+      };
 
-    panelRef = $mdPanel.create(config);
-    panelRef.open();
-    $rootScope.$apply();
+      panelRef = $mdPanel.create(config);
+      panelRef.open();
+      $rootScope.$apply();
 
-    var panelEl = document.querySelector('#panel');
-    expect(panelEl.parentElement).toBe(parentEl);
+      var panelEl = document.querySelector(PANEL_EL);
+      expect(panelEl.parentElement).toBe(parentEl);
 
-    panelRef.close();
-    $rootScope.$apply();
+      panelRef.close();
+      $rootScope.$apply();
+
+      expect(parentEl.childElementCount).toEqual(0);
+    });
+
+    it('should add md-panel if autoWrap = true', function() {
+      var template = '<div id="panel">Hello World!</div>';
+      var config = {
+        template: template,
+        autoWrap: true
+      };
+
+      panelRef = $mdPanel.create(config);
+
+      expect('md-panel').not.toExist();
+      expect('#panel').not.toExist();
+
+      panelRef.open();
+      $rootScope.$apply();
+
+      expect('md-panel').toExist();
+      expect('#panel').toExist();
 
-    expect(parentEl.childElementCount).toEqual(0);
+      panelRef.close();
```

Comment:
2 ideas:
- do `panelRef.close()` in an afterEach, so you don't have to include it in tests that don't care.
- extract these last lines into a new test, `it should remove panel from the dom on close` or somesuch, and remove from these two tests.